src/app/screens/BookDetails/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import i18next from 'i18next';
+import { useParams, useHistory } from 'react-router-dom';
+
+import backArrow from '~assets/images/back-arrow.png';
+import useAPI from '~hooks/useAPI';
+import { ENDPOINTS } from '~constants/api';
+import Loader from '~components/Loader';
+
+import Description from './components/Description';
+import styles from './styles.module.scss';
+
+export default function BookDetails() {
+  const { id } = useParams();
+  const history = useHistory();
+  const [{ isLoading, response }] = useAPI(
+    {
+      url: ENDPOINTS.bookDetails.replace(':id', id),
+      method: 'GET'
+    },
+    { data: {} },
+    true
+  );
+
+  const { author, editor, genre, imageUrl, title, year } = response.data;
```

Comment:
Qué pasa si la respuesta falla, o el server se cae?  está seguro que este componente no se va a romper?

Reply:
not because I'm giving it an initial value.

but I had not covered the case that I tried to access a reference that did not exist. So I added not found page